docs/building-apps/wallet/sep30.mdx

```diff
@@ -0,0 +1,301 @@
+---
+title: Recovery
+sidebar_position: 60
+---
+
+import Header from "./component/header.mdx";
+import { WalletCodeExample as CodeExample } from "@site/src/components/WalletCodeExample";
+
+<Header WIPLangs={["dart"]} />
+
+The [Sep-30] standard defines the standard way for an individual (e.g., a user or wallet) to regain access to their Stellar account after losing its private key without providing any third party control of the account. During this flow the wallet communicates with one or more recovery signer servers to register the wallet for a later recovery if it's needed.
+
+## Create Recoverable Account
+
+First, let's create an account key, a device key, and a recovery key that will be attached to the account.
+
+<CodeExample>
+
+```typescript
+const accountKp = wallet.stellar().account().createKeypair();
+const deviceKp = wallet.stellar().account().createKeypair();
+const recoveryKp = wallet.stellar().account().createKeypair();
+```
+
+```kotlin
```

Comment:
@Ifropc I copied over the kotlin example code from the [kotlin sdk wallet guide](https://github.com/stellar/kotlin-wallet-sdk/blob/3b81495e021939153d7f713a666b081ebbe6a61c/docs/WalletGuide.md#recovery), not sure if it's still up to date if you could take a look please 🙏

Reply:
It should be up to date